src/index.js

```diff
@@ -6,6 +6,8 @@ const { pushView } = require('./utils')
 
 module.exports = async function (req, res) {
   const { pathname, query } = url.parse(req.url, /* parseQueryString */ true)
+  res.setHeader('Access-Control-Allow-Origin', '*')
+  res.setHeader("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept")
```

Comment:
If I understand this right (correct me if I'm wrong), the `Access-Control-Allow-Headers` header tells the client which headers it can send and the server will respect, right? But we don't respect `Content-Type` nor `Accept` at all, we always just send JSON?